REQUIRE

```diff
@@ -0,0 +1,2 @@
+julia 0.3
+Compat
```

Comment:
Compat v0.7.15 has updated for `String`.
but that version of Compat was using the deprecated unix_only macro.
to declare with Compat v0.8.0 works fine.